src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -242,6 +246,168 @@ namespace VariableSpeedCoils {
 		VarSpeedCoil.deallocate();
 	}
 
+	// Default Constructor
```

Comment:
the default constructor is put here due to needing some pre-defined constant for initialization, like int const MaxSpedLevels( 10 ); // Maximum number of speed that supports
This will make it easy to increase the upper speed level later.